### PR TITLE
Fix bugs from chat-plugins refactor

### PR DESCRIPTION
**Pull request by CheeseMuffin**

### CODE REVIEW

File: chat-plugins/datasearch.js [hunk 1]
```diff
@@ -656,7 +656,7 @@ function runDexsearch(target, cmd, canAll, message) {
 	searches.sort((a, b) => Object.values(a).reduce(accumulateKeyCount, 0) - Object.values(b).reduce(accumulateKeyCount, 0));
 
 	let lsetData = {};
-	for (const group of searches) {
+	for (const group in searches) {
```
Comment (whalemer) [inline chat-plugins/datasearch.js]: Can't this just be ``(const alts of searches)``, since ``alts`` is only just the entry

Comment (Zarel) [reply chat-plugins/datasearch.js]: This should actually be `for (const alts of searches) {`

File: chat-plugins/datasearch.js [hunk 2]
```diff
@@ -656,7 +656,7 @@ function runDexsearch(target, cmd, canAll, message) {
 	searches.sort((a, b) => Object.values(a).reduce(accumulateKeyCount, 0) - Object.values(b).reduce(accumulateKeyCount, 0));
 
 	let lsetData = {};
-	for (const group of searches) {
+	for (const alts of searches) {
 		let alts = searches[group];
```
Comment (Zarel) [inline chat-plugins/datasearch.js]: ...ahem

Comment (whalemer) [inline chat-plugins/datasearch.js]: ...and then get rid of the line defining alts again

Comment (CheeseMuffin) [reply chat-plugins/datasearch.js]: ... i'm dumb

**Comment by Zarel** [comment]
Anyway there's no need for this; I'm doing the thing server-side...

**Comment by CheeseMuffin** [comment]
oh ok

**Pull request closed without merging**